test/unit/math/mathOps/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2017-2019 Benjamin Worpitz, Axel Huebl, Matthias Werner
+# Copyright 2017-2019 Benjamin Worpitz, Jakob Krude
```

Comment:
I assume this is accidental? 
(We never remove people from the copyright line, only add.)

Reply:
It was a copy-paste mistake in the beginning.

Reply:
Ah, thanks for clarifying.